src/components/Button/button.scss

```diff
@@ -0,0 +1,33 @@
+@import './variables';
+
+
+.btn {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 25px 80px;
+    background: $btn-color;
+    box-shadow: $box-shadow-btn-cateogry;
+    border-radius: 10px;
+    margin: 45px 0px;       
+    width: 255px;
+    height: 39px;
+    left: 80px;
+    top: 25px;
+
+    //text style
+    font-family: Inter;
+    font-style: normal;
+    font-size: 1em;
+    line-height: 39px;
+    text-transform: uppercase;
+    color: $btn-txt-color;
+    text-shadow: $text-shadow;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
1. Button is not responsive, it should be smaller on mobile screen
2. Use defined mixin instead of yout own media query

Reply:
Check it out, I'm not sure if it's ok now